assets/src/scripts/monitoring-location/components/daily-value-hydrograph/graph-controls.js

```diff
@@ -0,0 +1,89 @@
+import _ from 'lodash';
+import {select} from 'd3-selection';
+import {Actions} from '../../store/daily-value-time-series';
+
+/*
+ * Draw the parameter code toggle for the daily values time series graph.
+ * @param {Object} elem - D3 selection
+ * @param {Object} store - Redux Store
+ */
+const drawDVTimeSeriesSelection = function(ul, store) {
+
+    const availTimeSeries = store.getState()
+        .dailyValueTimeSeriesData.availableDVTimeSeries;
+    const groupedTs = _.groupBy(availTimeSeries, (ts) => {
+        return ts.parameterCode;
+    });
+    const groupedKeys = Object.keys(groupedTs);
+    let checked = false;
+
+    if (groupedKeys.length > 1) {
+        let divSize = 30;
+        ul.append('li')
+            .text('Parameter Codes');
+
+        groupedKeys.forEach((key) => {
+
+            const paramCd = groupedTs[key][0].parameterCode;
+            const ts_id = groupedTs[key][0].id.split('-')[2];
+            const monitorLoc = `${groupedTs[key][0].id.split('-')[0]}-${groupedTs[key][0].id.split('-')[1]}`;
+            const element = ul.append('li')
+                 .classed('usa-radio', true);
+
+            const input = element.append('input')
+                .classed('usa-radio__input', true)
+                .attr('type', 'radio')
+                .attr('id', `code-${paramCd}-radio`)
+                .attr('aria-labelledby', `code-${paramCd}-label`)
+                .attr('ga-on', 'click')
+                .attr('ga-event-category', 'DVSeriesGraph')
+                .attr('ga-event-action', 'toggleParameterCode')
+                .attr('name', 'dvParamCd');
+
+            input.on('click', function() {
+                store.dispatch(Actions.retrieveDVTimeSeries(
+                    monitorLoc, ts_id
+                )).then(function () {
+                    store.dispatch(Actions.setCurrentDVTimeSeriesIds(
+                        input.attr('data-00002'),
+                        input.attr('data-00003'),
+                        input.attr('data-00001')));
+                });
+            });
+
+            if (!checked){
```

Comment:
It might be clearer if you just initialize the first radio button to be checked, in the forEach you can use the second parameter which is the array index) to set the checked attribute only for the first input. So:
```if (index === 0) {
   input.attr('checked', 'checked')
}```

Reply:
I changed this to read idx == 0 before applying the property.